ansible/library/port_alias.py

```diff
@@ -69,27 +77,31 @@ def get_platform_type(self):
                     return value
         return None
 
-    def get_portconfig_path(self):
+    def get_portconfig_path(self, asic_id=None):
```

Comment:
Can we use the get path API already present in py-common since we are importing it ? https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/device_info.py#L221

Reply:
port_alias script is invoked from localhost i.e. from sonic-mgmt docker, when bringing up VS.
When running on localhost, we will not have access to sonic_py_common directory.
So we will still have to write a function to get the portconfig path.